packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -393,6 +393,8 @@ class AppDomain extends Domain {
 
     if (options.debuggingEnabled) {
       connectionInfoCompleter = new Completer<DebugConnectionInfo>();
+      // Callback hookups' Futures are uninteresting.
+      // ignore: unawaited_futures
       connectionInfoCompleter.future.then<Null>((DebugConnectionInfo info) {
```

Comment:
see earlier comments; the ignore should be on the same line, and the comment should be more specific. In particular in this case it's not at all clear to me that this future isn't interesting. What if e.g. _sendAppEvent throws? We'll end up with an uncaught error.